packages/api/src/schema/api-schema.yaml

```diff
@@ -859,12 +883,18 @@ components:
           type: string
           example: >-
             https://livepeercdn.com/asset/ea03f37e-f861-4cdd-b495-0e60b6d753ad/index.m3u8
-          description: URL for HLS playback
+          description: >-
+            URL for HLS playback. **It is recommended to not use this URL**, and
+            instead use playback IDs with the Playback Info endpoint to retrieve
+            the playback URLs - this URL format is subject to change.
```

Comment:
An example of the URL would be useful here.

Reply:
Done